util/FrontMatter.ts

```diff
@@ -26,4 +26,7 @@ export default interface FrontMatter {
     /** Image URL used for social image tag (og:image). If not present, the
      * social image defaults to the `image` field. */
     socialImage?: string
+
+    /** Content type */
+    type: string
```

Comment:
Just thinking ahead - we could eventually narrow this type (along with corresponding validation in the future) to: 
```ts
type: "posts" | "guides" // etc
```